File: Exercises_C++/1072.cpp
#include <iostream>
using namespace std;

int main() {

    int numero;
    cin >> numero;

    int count,dentro,fora,x;
    count = 0;
    dentro = 0;
    fora = 0;
    while (count < numero) {
        cin >> x;
        if (x >= 10 and x <= 20) {
            dentro += 1;
        }
        else {
            fora += 1;
        }
        count += 1;
    }

cout << dentro << " in" << endl;
cout << fora << " out" << endl;

}

File: Bee-Crowd/Exercises_Python/1072.py
nmr_casos = int(input())

numeros = []
for count in range(0,nmr_casos):
    numeros.append(int(input()))

dentro = []
fora = []
for c in numeros:
    if (c >= 10) and (c <= 20):
        dentro.append(c)
    else:
        fora.append(c)

print(len(dentro),'in')
print(len(fora),'out')